GeodeSupport.cmake

```diff
@@ -16,7 +16,7 @@ function(add_python_module _name)
   target_link_libraries(
     ${_name}
     PUBLIC
-      ${PYTHON_LIBRARIES}
+      ${PYTHON_LIBRARY}
```

Comment:
For consistency with cmake convention, can this be LIBRARIES instead of LIBRARY?